svelte-kit-scss/setupTests.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import matchers from '@testing-library/jest-dom/matchers';
+import { expect, vi } from 'vitest';
+import type { Navigation, Page } from '@sveltejs/kit';
+import { readable } from 'svelte/store';
+import * as environment from '$app/environment';
+import * as navigation from '$app/navigation';
+import * as stores from '$app/stores';
+
+// Add custom jest matchers
+expect.extend(matchers);
+
+// Mock SvelteKit runtime module $app/environment
+vi.mock('$app/environment', (): typeof environment => ({
+  browser: false,
+  dev: true,
+  building: false,
+  version: 'any',
+}));
+
+// Mock SvelteKit runtime module $app/navigation
+vi.mock('$app/navigation', (): typeof navigation => ({
+  afterNavigate: () => {},
```

Comment:
I think if you just called `vi.mock('$app/navigation')` you would get the same result

Reply:
Let me give it a try

Reply:
I believe this is necessary as some navigation types need to resolve async and others do not